src/containers/TweetsWithReplies.tsx

```diff
@@ -0,0 +1,26 @@
+import SingleTweet from "../components/SingleTweet";
+import tweets from "../MOCKS/tweets";
+import InfiniteList from "./InfiniteList";
+
+export const TweetsWithReplies: React.FunctionComponent = () => {
+  return (
+    <InfiniteList
+      data={tweets}
+      showMore={() => {}}
+      itemComponent={(itemData) => (
+        <SingleTweet
+          id={""}
+          userlogo={""}
+          username={""}
+          userNickname={""}
+          tweetText={""}
+          tweetDate={""}
+          retweetCount={0}
+          tweetQuoteCount={0}
+          likeCount={0}
+          {...itemData}
```

Comment:
I think it's a wrong way, because you're not providing proper type for item. Check [this for example](https://github.com/attech-org/leleka/blob/develop/src/containers/ProfileMedia.tsx#L15)